### PR TITLE
Interchain Exchange review

### PR DESCRIPTION
**Pull request by toschdev**

### CODE REVIEW

File: interchain-exchange/tutorial/00-intro.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ order: 1
 # Introduction
 
 The Interchain Exchange is a module to create buy and sell orders in between blockchains.
-In this tutorial you will learn how to create a Cosmos SDK module which can create order pairs, buy and sell orders. You will be able to create orderbooks, buy and sell orders across blockchains, which enables to swap tokens from one blockchain to another.
+In this tutorial you will learn how to create a Cosmos SDK module which can create order pairs, buy and sell orders. You will be able to create orderbooks, buy and sell orders across blockchains.
```
Comment (lubtd) [inline interchain-exchange/tutorial/00-intro.md:8]:
```suggestion
In this tutorial, you will learn how to create a Cosmos SDK module that can create order pairs, buy and sell orders. You will be able to create order books, buy and sell orders across blockchains.
```

Comment (toschdev) [reply interchain-exchange/tutorial/00-intro.md:8]: We always use `orderbook` as one-word. I would like to stay consistent with this wording.

Comment (lubtd) [reply interchain-exchange/tutorial/00-intro.md:8]: Shouldn't we change it everywhere in the future? I mean I don't think orderbook is grammatically correct

Comment (toschdev) [reply interchain-exchange/tutorial/00-intro.md:8]: We can use `order book` and then consistent. I can make a PR with that change tomorrow

File: interchain-exchange/tutorial/00-intro.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ order: 1
 # Introduction
 
 The Interchain Exchange is a module to create buy and sell orders in between blockchains.
-In this tutorial you will learn how to create a Cosmos SDK module which can create order pairs, buy and sell orders. You will be able to create orderbooks, buy and sell orders across blockchains, which enables to swap tokens from one blockchain to another.
+In this tutorial you will learn how to create a Cosmos SDK module which can create order pairs, buy and sell orders. You will be able to create order books, buy and sell orders across blockchains.
```
Comment (lubtd) [inline interchain-exchange/tutorial/00-intro.md:8]: `which enables to swap tokens from one blockchain to another` why removing this statement? This was pretty cool

**Review by lubtd — APPROVED** [review]

File: interchain-exchange/tutorial/00-intro.md (hunk 2 above)
Comment (toschdev) [reply interchain-exchange/tutorial/00-intro.md:8]: Ok, great. I thought it might create other expectations. But if you approve I love to keep it 😁

Comment (lubtd) [reply interchain-exchange/tutorial/00-intro.md]: Well it's actually about token swap

**Review by barriebyron — APPROVED** [review]
approving without thorough review, will revisit for editorial details next round, good job!

**Pull request merged**